File: app/src/main/java/net/osdn/gokigen/a01d/camera/panasonic/wrapper/eventlistener/ICameraStatusHolder.java
package net.osdn.gokigen.a01d.camera.panasonic.wrapper.eventlistener;

import java.util.List;

public interface ICameraStatusHolder
{
    String getCameraStatus();
    boolean getLiveviewStatus();
    String getShootMode();
    List<String> getAvailableShootModes();
    int getZoomPosition();
    String getStorageId();

}

File: app/src/main/java/net/osdn/gokigen/a01d/camera/panasonic/wrapper/eventlistener/ICameraStatusHolder.kt
package net.osdn.gokigen.a01d.camera.panasonic.wrapper.eventlistener

interface ICameraStatusHolder
{
    fun getCameraStatus(): String
    fun getLiveviewStatus(): Boolean
    fun getShootMode(): String
    fun getAvailableShootModes(): List<String>
    fun getZoomPosition(): Int
    fun getStorageId(): String
}